components/feature/toolbar/src/main/java/mozilla/components/feature/toolbar/ToolbarFeature.kt

```diff
@@ -7,26 +7,31 @@ package mozilla.components.feature.toolbar
 import mozilla.components.concept.toolbar.Toolbar
 import mozilla.components.browser.session.SessionManager
 
+typealias LoadUrlUseCase = (String) -> Unit
 /**
  * Feature implementation for connecting a toolbar implementation with the session module.
  */
 class ToolbarFeature(
     sessionManager: SessionManager,
-    toolbar: Toolbar
+    loadUrlUseCase: LoadUrlUseCase,
```

Comment:
This part looks like in my prototype again. Just that this is a function here. :)